.github/workflows/sparse-zephyr.yml

```diff
@@ -61,8 +61,9 @@ jobs:
                west update --narrow --fetch-opt=--depth=5
 
       # Not strictly necessary but saves a lot of scrolling in the next step
-      # TODO, research caching:
-      # https://stackoverflow.com/questions/66421411/how-to-run-cached-docker-image-in-github-action
+      # Caching a 12G image is unfortunately not possible:
```

Comment:
Why is the image 12G ? Are we carrying stuff we dont need anymore like old GCC versions for legacy HW ?